modules/setting/lfs.go

```diff
@@ -13,6 +13,7 @@ import (
 // LFS represents the configuration for Git LFS
 var LFS = struct {
 	StartServer    bool          `ini:"LFS_START_SERVER"`
+	AllowPureSSH   bool          `ini:"LFS_ALLOW_PURE_SSH"`
```

Comment:
I would prefer on by default or has it some drawbacks?

Reply:
It's disabled by default as the user could run into the same issue that you did if they haven't configured their client. We can change it to true once the patch to LFS has been merged and it has made its way to OS packages

Reply:
well in my case it just used http(s) as fallback ... so i do not see why it would harm

Reply:
We'd want to wait until the upstream git-lfs changes are merged and rolled out to end distributions so that they don't run into what that PR is trying to solve without being aware of the implications of enabling it right now.